MHVLibTests/Clients/MHVThingClientBlobTests.m

```diff
@@ -0,0 +1,502 @@
+//
+// MHVThingClientBlobTests.m
```

Comment:
Updated all the tests to do the shouldEventually() and do check results instead of internal states.